app/jobs/onceoff/rename_canned_replies.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module Jobs
-  class RenameCannedReplies < Jobs::Onceoff
+  class RenameCannedReplies < ::Jobs::Onceoff
```

Comment:
just confirming... does ` class RenameCannedReplies < Onceoff` work?  If not ... why not?